content/cpp/concepts/vectors/terms/size/Capacity/Capacity.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: '.capacity()'
+Description: 'Returns the storage space currently allocated for the vector'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Vectors'
+  - 'Arrays'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **.capacity()** method returns the storage space allocated for a vector. This is different from .size(). Capacity can be equal or greater than size. It is the total number of elements that a vector can hold without reallocation. This is automatically assigned at initialization.
+
+## Syntax
+
+The `.capacity()` method requires no parameters and can be called as follows
+
+```pseudo
+myvector.capacity()
+```
+
+## Example
+
+The following code initializes the vectors and returns its capacity
+
+```codebyte/cpp
```

Comment:
```suggestion
```cpp
```

Reply:
Since this is an example, we will use the `cpp` block. For ##Codebyte Example blocks, we use the `codebyte/cpp`